addons/calendar/data/mail_template_data.xml

```diff
@@ -22,7 +22,12 @@
         Hello <t t-out="object.common_name or ''">Wood Corner</t>,<br/><br/>
 
         <t t-if="not target_responsible">
-            <t t-out="object.event_id.user_id.partner_id.name or ''">Colleen Diaz</t> invited you for the <strong t-out="object.event_id.name or ''">Follow-up for Project proposal</strong> meeting.
+            <t t-if="object.event_id.user_id.partner_id.id == object.env.ref('base.partner_root').id">
```

Comment:
@amdi-odoo Could it not exist in some dbs? Maybe we could add raise_if_not_found=False in ref() to be sure we do not have an issue here? If it always exist, looks ok as is :)